docs/content/docs/connectors/table/filesystem.md

```diff
@@ -438,4 +444,43 @@ FROM kafka_table;
 SELECT * FROM fs_table WHERE dt='2020-05-20' and `hour`='12';
 ```
 
+```sql
```

Comment:
Please add a short description (what's the purpose) for the following example. The **Full Example** is not just for your new introduced config option, most users don't have the context.